pkg/sql/crdb_internal.go

```diff
@@ -644,6 +673,7 @@ CREATE TABLE crdb_internal.session_trace (
   message     STRING NOT NULL,     -- The logged message.
   age         INTERVAL NOT NULL    -- The age of this message relative to the beginning of the trace.
 )`,
+	comment: "latest trace collected on this session (via SET TRACING={ON/OFF}).",
```

Comment:
nit/style: when we opt out of full sentences (no capital at start) we also opt out of the final period. (Same as cluster settings)

Conversely you could opt into the capitals.